docs/PacketFence_Rapid7NexposeInsightVM_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,185 @@
+Rapid7 InsightVM Nexpose: Quick Integration Guide
+-------------------------------------------------
+:doctype: book
+:toc: left
+
+include::includes/global-attributes.asciidoc[]
+
+////
+
+    This file is part of the PacketFence project.
+
+    See PacketFence_Administration_Guide-docinfo.xml for authors, copyright 
+    and license information.
+
+////
+
+About this Guide
+----------------
+
+This guide has been created in order to help the configuration of Rapid7 InsighVM Nexpose and PacketFence, to provide informations and actions about devices compliances before and during network access.
```

Comment:
This guide has been created in order to help you with the configuration of Rapid7 InsighVM Nexpose and PacketFence, to provide information and actions about devices compliance before and during network access.